test/integration/battery_plugin.cc

```diff
@@ -39,18 +39,17 @@
 #include "ignition/gazebo/components/Name.hh"
 
 #include "plugins/MockSystem.hh"
+#include "../helpers/EnvTestFixture.hh"
 
 using namespace ignition;
 using namespace gazebo;
 
-class BatteryPluginTest : public ::testing::Test
+class BatteryPluginTest : public InternalFixture<::testing::Test>
 {
   // Documentation inherited
   protected: void SetUp() override
   {
-    common::Console::SetVerbosity(4);
-    ignition::common::setenv("IGN_GAZEBO_SYSTEM_PLUGIN_PATH",
-           (std::string(PROJECT_BINARY_PATH) + "/lib").c_str());
+    InternalFixture::SetUp();
```

Comment:
If we call `SetUp` but don't call `TearDown`, does that mean the `IGN_HOMEDIR` environment variable doesn't get reset correctly?

Reply:
We're overriding the `SetUp` function, so we explicitly call the base class's `SetUp` to make sure it runs. Since we're not overriding `TearDown`, the base class's function will be called.